backends/ida/graffiti.py

```diff
@@ -235,7 +238,28 @@ def activate(self, ctx):
     def update(self, ctx):
         return idaapi.AST_ENABLE_ALWAYS
 
-def sync_read_thread():
+
+def bring_ida_to_foreground():
+    # according to https://hex-rays.com/blog/plugin-focus-heimdallr/
+    # Tested only on macos, should be adapted easily to Linux/Windows
+    if sys.platform == 'darwin':
+    # https://www.riverbankcomputing.com/static/Docs/PyQt5/
+        qtwidget = ida_kernwin.PluginForm.TWidgetToPyQtWidget(ida_kernwin.get_current_viewer())
+        window = qtwidget.window()
+
+        # UnMinimize
+        WindowMinimized = 0x00000001  # https://www.riverbankcomputing.com/static/Docs/PyQt5/api/qtcore/qt.html#WindowState
+        cur_state = window.windowState()
+        new_state = cur_state & (~WindowMinimized)
+        window.setWindowState(new_state)
+
+        # Switch desktop / give keyboard control
+        window.show()
+        window.raise_()  # Bring to front (MacOS)
```

Comment:
Support windows part 2
```suggestion
        if sys.platform == 'darwin':
            window.raise_()
        elif sys.platform == 'win32':
            window.activateWindow()
```